libs/service-portal/applications/src/lib/messages.ts

```diff
@@ -94,4 +94,24 @@ export const m = defineMessages({
     description:
       'Error message when no applications are found in the overview for national id',
   },
+  noIncompleteApplicationsAvailable: {
+    id: 'sp.applications:no-incomplete-applications-available',
+    defaultMessage: 'Engar ókláraðar fundust hjá Ísland.is á þessari kennitölu',
```

Comment:
Is the word "umsóknir" missing inbetween "engar ókláraðar - fundust" ?

Reply:
Fixed, thanks!